control/cli.py

```diff
@@ -166,13 +166,15 @@ def delete_bdev(self, args):
     @cli.cmd([
         argument("-n", "--subnqn", help="Subsystem NQN", required=True),
         argument("-s", "--serial", help="Serial number", required=True),
+        argument("-m", "--maxns", help="Maximum number of namespaces", type=int, default=0, required=False),
```

Comment:
I thought we agreed about "--max-namespaces" name for the option?

Reply:
@trociny @idryomov addressed this now. Please have a look